pkg/store/sandbox/sandbox.go

```diff
@@ -19,6 +19,7 @@ package sandbox
 import (
 	"sync"
 
+	"github.com/containerd/containerd"
```

Comment:
nit: Maybe too picky, but may help with future PRs, :p 
```
import(
"sync" // golang libraries
// empty line
"github.com/containerd/containerd" // other libraries
// empty line
"github.com/kubernetes-incubator/cri-containerd/pkg/store" // CRI containerd libraries
)
```

And also the same with the other imports.